src/pages/listvideos/listvideos.css

```diff
@@ -0,0 +1,69 @@
+.title-section{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+}
+.video-container{
+    display: grid;
+    grid-template-columns: 1fr 4fr;
+    gap: 20px;
+    align-items: start;
+    padding-top: 2rem;
+    padding: 3rem;
+}
+.video-listing, .menu-sidebar{
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+}
+
+.grid-title{
+    font-size: 2rem;
+    font-weight: 600;    
+}
+.hr-div-long{
+    border: 1px solid black;
+    width: 500px;
+}
+.hr-div-short{
+    border: 1px solid black;
+    width: 250px;
```

Comment:
avoid using absolute units if it exceedes 15px , use reletive units like rem or even %

Reply:
understood, will follow